sk1b.csv

```diff
@@ -0,0 +1,1489 @@
+,SV,time,geo,observation
```

Comment:
Can you please confirm these files are (sk1b.csv, sk1c.csv ....).
Also please ensure no index is written to output files.

Reply:
Done with the changes.